src/client/src/actions/wizardContentActions/getProjectTypes.ts

```diff
@@ -6,29 +6,6 @@ import getSvgUrl from "../../utils/getSvgUrl";
 import WizardContentActionType from "./wizardContentActionType";
 import { Dispatch } from "react";
 
-// thunk
```

Comment:
Do you need this action at all then?

Reply:
@SahilTara you're right. 
Honestly, I'm not sure if this file is needed- The methods are not used at all, but the reason I did not remove it is because we don't know if there will be other project types added in the future and

Reply:
@Tanya0609 did you implement getting project types from the extension? If so you should update this file. We should keep the action in case there are more project types in the future as Jasmin mentioned.